src/Featurizers/StringFeaturizer.h

```diff
@@ -88,9 +137,85 @@ class StringEstimator : public Components::InferenceOnlyEstimatorImpl<StringTran
 // |  StringTransformer
 // |
 // ----------------------------------------------------------------------
+template <typename T>
+StringTransformer<T>::StringTransformer(bool useEmptyStringsForNullValues /*=false*/) :
+    _useEmptyStringsForNullValues(
+        std::move(
+            [&useEmptyStringsForNullValues](void) -> bool & {
```

Comment:
Since we just talked about reference capture in lambdas, just want to make sure I understand right. this one is ok because the lambda is part of the class and so is the boolean value, is that correct?

Reply:
The lambda is invoked immediately, so the lifetime of the referenced value is greater than the lifetime of the lambda.